src/deluge/gui/l10n/english.cpp

```diff
@@ -747,17 +747,10 @@ PLACE_SDRAM_DATA Language english{
 
```

Comment:
re-grouped automation view strings together at the bottom to make for a more clean-up presentation.